integration-tests/src/test/resources/indexer/wikipedia_merge_index_task.json

```diff
@@ -0,0 +1,70 @@
+{
+    "type": "index",
+    "spec": {
+        "dataSchema": {
+            "dataSource": "%%DATASOURCE%%",
+            "metricsSpec": [
+                {
+                    "type": "count",
+                    "name": "count"
+                },
+                {
+                    "type": "doubleSum",
+                    "name": "added",
+                    "fieldName": "added"
+                },
+                {
+                    "type": "doubleSum",
+                    "name": "deleted",
+                    "fieldName": "deleted"
+                },
+                {
+                    "type": "doubleSum",
+                    "name": "delta",
+                    "fieldName": "delta"
+                },
+                {
+                    "type": "stringFirst",
+                    "name": "first_user",
+                    "fieldName": "user"
+                },
+                {
+                    "type": "stringLast",
+                    "name": "last_user",
+                    "fieldName": "user"
+                }
```

Comment:
Can we add these to `wikipedia_index_task.json` instead. This way we don't need to run another integration test which can be quite slow

You will probably want to do something very similar to #9277

Reply:
The issue described in #7243 requires a trigger of merging of two index files, which will in turn cause a rollup of two rows. I wasn't able to trigger that in `wikipedia_index_task.json`. This is why I added a new task to capture this case.

Reply:
I also added these in `wikipedia_index_task.json`, which led me to discover the issue with `StringFirstAggregatorFactory.combine()` below.